### ISSUE TITLE
loss.backward() dimension issue on M1 chip

### ISSUE THREAD

**Issue by att-ar** [att-ar/pytorch_colab#1]
The backward pass throws an error when I run the train loop on my M1 Macbook. But it works perfectly fine on Google Colab's Cuda GPUs.

**Comment by att-ar** [comment]
It is the Huber Loss issue I think. The HuberLoss() does not support Apple Metal.
